### PR TITLE
Change Freckle to Noko in blog post

### PR DESCRIPTION
**Pull request by FionaDL**
This PR updates the blog post on Pecas so that Freckle is now exchanged for Noko. 
https://www.pivotaltracker.com/story/show/173803668

### CODE REVIEW

File: _posts/2016-06-06-introducing-pecas.markdown [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-title: "Introducing Pecas: Dashboards for Freckle"
+title: "Introducing Pecas: Dashboards for Noko"
 date: 2016-06-06 9:31:00
-reviewed: 2020-03-05 10:00:00
+reviewed: 2020-09-15 10:00:00
 categories: ["open-source"]
 author: "etagwerker"
 ---
 
 At [Ombu Labs](https://www.ombulabs.com) we are big fans and happy customers
```
Comment (etagwerker) [inline _posts/2016-06-06-introducing-pecas.markdown:10]: @FionaDL While you are at it, please update this to OmbuLabs

File: _posts/2016-06-06-introducing-pecas.markdown [hunk 2]
```diff
@@ -32,8 +32,8 @@ good way to detect deviations.
 
 To build this small application we decided to use
 [Rails 4.2](http://rubyonrails.org/) and the
```
Comment (etagwerker) [inline _posts/2016-06-06-introducing-pecas.markdown:34]: @FionaDL Is this still accurate? We should probably create an issue (or set of issues) to upgrade this to Rails 6.0

File: _posts/2016-06-06-introducing-pecas.markdown [hunk 3]
```diff
@@ -32,8 +32,8 @@ good way to detect deviations.
...
-[Freckle API v2](http://developer.letsfreckle.com/#overview). You can easily
-set it up using [Heroku](http://heroku.com) and your Freckle API key: [https://github.com/ombulabs/pecas#first-time-only](https://github.com/ombulabs/pecas#first-time-only)
+[Noko API v2](https://developer.nokotime.com/). You can easily
+set it up using [Heroku](http://heroku.com) and your Noko API key: [https://github.com/ombulabs/pecas#first-time-only](https://github.com/ombulabs/pecas#first-time-only)
 
-If you like [Freckle](https://letsfreckle.com) as much as we do, feel free to
+If you like [Noko](https://nokotime.com/) as much as we do, feel free to
 **fork Pecas** in [Github](https://github.com/ombulabs): [https://github.com/ombulabs/pecas](https://github.com/ombulabs/pecas)
```
Comment (etagwerker) [inline _posts/2016-06-06-introducing-pecas.markdown:39]: @FionaDL Repo is now under the fastruby organization.

**Review by etagwerker — COMMENTED** [review]
@FionaDL Thanks for taking the initiative on this! Just added a couple of comments. 💯

File: _posts/2016-06-06-introducing-pecas.markdown (hunk 3 above)
Comment (FionaDL) [reply _posts/2016-06-06-introducing-pecas.markdown:39]: @erGood call!

> @FionaDL Repo is now under the fastruby organization.

@etagwerker good call!

File: _posts/2016-06-06-introducing-pecas.markdown (hunk 2 above)
Comment (FionaDL) [reply _posts/2016-06-06-introducing-pecas.markdown:34]: looks like we are still using Rails 4.2, I will add an issue.

Comment (FionaDL) [reply _posts/2016-06-06-introducing-pecas.markdown:34]: Looks like there is already an issue open for this: https://github.com/fastruby/pecas/issues/55

**Review by lubc — APPROVED** [review]
@FionaDL Looks good! Thanks!

**Pull request merged**